arrays/array_practice.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+
+# create an array to work with
+# that has integers 1-10
+play_array = (1..10).to_a
+
+# my observations while coding this:
+# puts play_array
+# puts play_array.class
+# print play_array
+# puts ''
+
+# Learnings:
+
+# surprised when later changes to dup_array affected orig_array
+# dup_array = orig_array
+# so this was the better way to duplicate arrays
+# dup_array = orig_array.take(10)
+
+# working, but tricky to chop the last comma:
+# print 'T-'
+# play_array.reverse_each {|x| print x, ', '}
+# print '... BLASTOFF!'
+
+# working, but better practice to place in string
+# print 'T-' + play_array.reverse.join(', ') + '... BLASTOFF!'
+
+# calculate stuff needed for the output lines
+blastoff_string = 'T-' + play_array.reverse.join(', ') + '...  BLASTOFF!'
+# blastoff_string2 = 'T-' + play_array.reverse.join(', ') + '...  BLASTOFF!'
+# blastoff_string3 = 'T-' + play_array.reverse.join(', ') + '... BLASTOFF!'
+
+index = 3
+first_four = play_array[0..3].join(', ')
+
+shortened_array = play_array.take(10)
+shortened_array.delete(5)
+shortened_array.delete(6)
+shortened_array.delete(7)
+
+rotated_array = shortened_array.take(7)
+rotated_array.push(5)
+rotated_array.rotate(-1)
```

Comment:
orig_array.unshift(5)  was supposed to, and did in irb.. but would not work from code file execution.

So I had to find another way to do it.